src/app/analyze/analyze-overview/analyze-overview.component.ts

```diff
@@ -11,26 +11,36 @@ import { ContextService } from '../../shared/context.service';
   templateUrl: 'analyze-overview.component.html',
   styleUrls: ['./analyze-overview.component.scss']
 })
-export class AnalyzeOverviewComponent {
+export class AnalyzeOverviewComponent implements OnInit {
 
   imgLoaded: Boolean = false;
 
   space: Space;
 
   constructor(
+    private route: ActivatedRoute,
     private router: Router,
     context: ContextService,
+    private spaceService: SpaceService,
     private broadcaster: Broadcaster
   ) {
-    context.current.subscribe(val => this.space = val.space);
+  }
+
+  ngOnInit() {
+    this.route.params
+      .switchMap((params: Params) => this.spaceService.getSpaceByName(params['entity'], params['space']))
+      .subscribe((space: Space) => this.space = space);
```

Comment:
Hah, I spoke too soon! We need to move an interface for ContextService over so that projects like planner can get at the context and space change subscriptions directly.